crates/egui/src/menu.rs

```diff
@@ -105,17 +105,17 @@ pub fn menu_button<R>(
     stationary_menu_impl(ui, title, Box::new(add_contents))
 }
 
-/// Construct a top level menu with an image in a menu bar. This would be e.g. "File", "Edit" etc.
+/// Construct a top level menu with a custom button in a menu bar.
 ///
 /// Responds to primary clicks.
 ///
 /// Returns `None` if the menu is not open.
-pub fn menu_image_button<R>(
```

Comment:
Let's keep the old `menu_image_button` as well, but `#[deprecated]`, and forward it to `menu_custom_button`, so we don't break users code.

You may need to make `ImageButton::image` `pub(crate)` in order to turn it into a `Button`.